backend/src/Designer/Views/Shared/Index.cshtml

```diff
@@ -4,5 +4,5 @@
     var jsPath = Url.Content($"/designer/frontend/{app}/{app}.js");
 }
 
-<link href="@cssPath" asp-append-version="true" rel="stylesheet">
```

Comment:
Without this "flag" `asp-append-version="true"`, would the browser still know which file is the latest one? I mean, I think this flag is adding a "random" string/identifier behind the static file, and this is used to inform about changes within those files. By removing this, does the files have any hash at all? 😊

Reply:
Ah okay, I don't know why I thought it was an old attribute that was no longer in use 🫤
So, there might be an issue with this tag as it doesn't seem to add the random string to our static filenames in production 🤔

Reply:
I reverted my changes. https://github.com/Altinn/altinn-studio/issues/12687 should solved the caching issue